edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/util/aggrigation/Aggrigators.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Util.Aggrigation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    public class Summer : IAggrigator
+    {
+        double sum = 0;
+
+        public IAggrigator New()
+        {
+            return new Summer();
+        }
+
+        public void PutValue(double value)
+        {
+            this.sum += value;
+        }
+
+        public double GetAggrigate()
+        {
+            return this.sum;
+        }
+    }
+
+    public class Multiplier : IAggrigator
```

Comment:
Multiplier isn't used in the product code. Should we remove it until we need it?

Reply:
I'm ambivalent. It's useful for the tests that check mixing aggregators, and its pretty short